StRoot/StETofUtil/StETofGeometry.cxx

```diff
@@ -1488,4 +1542,69 @@ StETofGeometry::getFieldZ( const double& x, const double& y, const double& z ) {
     mStarBField->BField( X, B );
 
     return B[ 2 ];
-}
\ No newline at end of file
+}
+
+void 
+StETofGeometry::readAlignmentParameters(){
+      LOG_INFO << "etofAlignParam: filename provided --> use parameter file: " << mFileNameAlignParam << endm;
+        
+		  //add check for no alignment set here.
+	
+		  ifstream paramFile;		
+        paramFile.open( mFileNameAlignParam );
+
+        if( !paramFile.is_open() ) {
+            LOG_INFO << "unable to get the alignments parameters from file --> file does not exist" << endm;
+            return;
+        }
+
+	     StThreeVectorD counterAlignmentParameter;
+        float tempX;
+		  float tempY;
+		  float tempZ;
+        while( paramFile >> tempX >> tempY >> tempZ ) {
+		  counterAlignmentParameter = StThreeVectorD( tempX, tempY, tempZ );
+            mAlignmentParameters.push_back( counterAlignmentParameter );
+        }
+        
+        paramFile.close();
+
+        if( mAlignmentParameters.size() != 108 ) {
+            LOG_INFO << "parameter file for alignments has not the right amount of entries: ";
+            LOG_INFO << mAlignmentParameters.size() << " instead of 108 !!!!" << endm;
+            return;
+        }
+
```

Comment:
If the number of values read from the file is not 108 as expected it'd better be a warning

```suggestion
        if( mAlignmentParameters.size() != 108 ) {
            LOG_WARN << "parameter file for alignments has not the right amount of entries: "
                     << mAlignmentParameters.size() << " instead of 108 !!!!" << endm;
        }
```